src/src/tx_main.cpp

```diff
@@ -390,6 +390,22 @@ void ICACHE_RAM_ATTR HandlePrepareForTLM()
   }
 }
 
+int8_t ICACHE_RAM_ATTR PowerEnumToLBTLimit(PowerLevels_e txPower)
+{
+    // Calculated from EN 300 328, assuming 800kHz BW for sx1280
+    // TL = -70 dBm/MHz + 10 × log10 (100 mW / Pout) (Pout in mW e.i.r.p.)
+    // Values above 100mW are not relevant, default to 100mW threshold
+    // TODO: This threshold should be modified with a config for antenna gain
+    switch(txPower)
+    {
+    case PWR_10mW: return -78;
```

Comment:
Should this not be 0 since LBT is not required at 10mW?

Reply:
I'm not 100% sure, see this section about the adaptive fhss clause:

> In addition, this requirement does not apply for FHSS equipment with a maximum declared RF Output power level of less than 10 dBm e.i.r.p. **or for FHSS equipment when operating in a mode where the RF Output power is less than 10 dBm e.i.r.p.**

I read that to mean the equipment can skip LBT if operating in a mode with strictly <10mW tx power at all times. But if the equipment Can operate with higher power, it is the higher power mode that will be tested for adaptive fhss/LBT behaviour, and if we send out a <10mW power packet at the wrong frequency during that test, it will fail the test. As I understand it...

Reply:
Hmm, yeah I think you are right. So when power is manually set to 10mW LBT can be disabled ben when dynamic power is enabled and power happens to be on 10mW we would still need to use the -78dBm threshold. And during certification testing the output power will be set to 100mW (I assume the firmware will need to be adjusted to make this max level) so the -88dBm threshold applies.